projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -864,7 +879,7 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
 
         if (range.length > 0) {
             this.calendar.selectDate(range);
-        } else {
+        } else if (range.length <= 0 && this.calendar.monthViews) {
```

Comment:
could this be less than 0
:grinning:
```suggestion
        } else if (range.length === 0 && this.calendar.monthViews) {
```